packages/extension/svelte-stuff/shared/io.ts

```diff
@@ -5,10 +5,7 @@ export const getSocket = () => {
   if (!socket) {
     socket = new ReconnectingWebSocket(
       () =>
-        (apiBaseUrl.includes("https")
-          ? apiBaseUrl.replace("https", "wss")
-          : apiBaseUrl.replace("http", "ws")) +
-        `?accessToken=${accessToken}&refreshToken=${refreshToken}`
+        `${apiBaseUrl.replace("http", "ws")}?accessToken=${accessToken}&refreshToken=${refreshToken}`
```

Comment:
What if the base url includes "https" instead of "http"? Please re-check this line

Reply:
@RazerMoon 
```javascript
 `https://some.host/api`.replace('http', 'ws'); // wss://some.host/api
```

Still should works fine, cause `https` and `wss` contain common last `s`.
Maybe I'm not correctly understand the question, let me know. Thanks !

Reply:
I didn't see that, you're right. My apologies 😅